WebDriverAgentTests/IntegrationTests/FBW3CTouchActionsIntegrationTests.m

```diff
@@ -301,6 +301,7 @@ - (void)testDoubleTap
           ],
       },
     ];
+  // Maybe the landscape cannnot handled well
```

Comment:
do we need these comments?

Reply:
Will remove such comments after meginr your PR :)